docs/common/yum.md

```diff
@@ -26,11 +25,7 @@ intended for public use:
     See the [upcoming software section](#upcoming-software) for details.
 
 !!! note
-    In the 3.5 series (and earlier), packages are added to the `osg` and `osg-upcoming` repositories
-    during discrete, numbered releases (e.g. 3.5.31);
-    packages are added to the `osg-rolling` and `osg-upcoming-rolling` repositories as soon as they are considered production-ready.<br>
-    In the 3.6 series, there are no `osg-rolling` or `osg-upcoming-rolling` repositories,
-    and packages are added to the `osg` and `osg-upcoming` repositories as soon as they are considered production ready.
+    In the 3.6 series, packages are added to the `osg` and `osg-upcoming` repositories as soon as they are considered production ready.
```

Comment:
I think we can just drop this admonition since the above table basically says this already

Reply:
I agree